### DISCUSSION TITLE
IPadapter and ControlNet : more  parameters

### DISCUSSION THREAD

**Discussion by K3nt3L** [lllyasviel/Fooocus#3420] [Ideas]
If possible, a modification of the IPadapter settings would be a real plus. Taking inspiration from @Cubiq 's work, it would be good to be able to specify two modes: composition and  style.
[GitHub IP adapter Plus](https://github.com/cubiq/ComfyUI_IPAdapter_plus)

It should also be possible to set the starting point (Start at), and not just the endpoint (Stop at) of the ControlNets and the IPadapter.
Being able to change the starting point would allow for much more creativity and control.

And more fun ;)

![fooocus_for_fun](https://github.com/user-attachments/assets/2092643f-2e27-4181-a286-16cee0910af3)

**Comment by mashb1t** [comment]
similar to this feature request: https://github.com/lllyasviel/Fooocus/issues/2648
duplicate of https://github.com/lllyasviel/Fooocus/issues/2800 / https://github.com/lllyasviel/Fooocus/discussions/2709
